src/app/tests/suites/certification/Test_TC_DRLK_2_2.yaml

```diff
@@ -219,22 +219,50 @@ tests:
           "TH sends Lock Door Command to the DUT without any argument PINCode"
       PICS: DRLK.S.C00.Rsp && DRLK.S.A0033
       command: "LockDoor"
-      timedInteractionTimeoutMs: 10000
+      timedInteractionTimeoutMs: 1000
       response:
           error: FAILURE
 
+    - label: "TH reads the WrongCodeEntryLimit attribute from the DUT"
+      PICS: DRLK.S.A0030
+      command: "readAttribute"
+      attribute: "WrongCodeEntryLimit"
+      constraints:
+          minValue: 1
+          maxValue: 255
+
+    - label: "TH sends the unlock Door command to the DUT with invalid PINCode"
+      PICS: DRLK.S.A0030
+      command: "UnlockDoor"
+      timedInteractionTimeoutMs: 1000
+      arguments:
+          values:
+              - name: "PINCode"
+                value: "1234568"
+      response:
+          error: FAILURE
+
+    - label: "TH sends the unlock Door command to the DUT with valid PINCode"
```

Comment:
Step 10c should be executed within `PIXIT.DRLK.UserCodeTemporaryDisableTime`, and expecting fail.
then 10d to be ran after `PIXIT.DRLK.UserCodeTemporaryDisableTime` expired, and expect success.